WalletWasabi/KeyManagement/KeyManager.cs

```diff
@@ -560,7 +560,7 @@ public ExtKey GetMasterExtKey(string password)
 		{
 			if (IsWatchOnly)
 			{
-				throw new SecurityException("This is a watchonly wallet.");
+				throw new SecurityException("This is a watch-only wallet.");
```

Comment:
Please don't change exception messages, it's dangerous. Also the spelling is widely used.